internal/v2/remote_signer/remote_signer_test.go

```diff
@@ -0,0 +1,41 @@
+package remotesigner
+
+import (
+	"testing"
+	"crypto"
+	"crypto/tls"
+	_ "embed"
+)
+
+var (
+	//go:embed example_cert_key/cert.pem
+	cert []byte
+	//go:embed example_cert_key/key.pem
+	key []byte
+)
+
+// TestNewSigner runs unit test for NewRemoteSigner.
+func TestNewSigner(t *testing.T) {
+	// Setup for static test.
+	cert0, err0 := tls.X509KeyPair(cert, key)
```

Comment:
nit: remove "0" suffix on variables, since we don't have multiple certs or errors.